.github/workflows/pytest.yml

```
@@ -131,6 +146,6 @@ jobs:
     # tmate. However, don't leave them uncommented on merge as that
     # causes failing builds to hang forever.
     #
-    # - name: tmate session
-    #   if: ${{ failure() }}
-    #   uses: mxschmitt/action-tmate@v3
+    - name: tmate session
+      if: ${{ failure() }}
+      uses: mxschmitt/action-tmate@v3
```

```suggestion
    # - name: tmate session
    #   if: ${{ failure() }}
    #   uses: mxschmitt/action-tmate@v3
```

just to make this easy to disable